cg-spec/editors_draft.html

```diff
@@ -601,25 +557,9 @@ <h2>Initial Definitions</h2>
     <section id="sparql-star-grammar">
       <h2>Grammar</h2>
 
-      <p>SPARQL-star is defined to follow the <a data-cite="SPARQL11-QUERY#sparqlGrammar">same grammar</a> as SPARQL 1.1, <em>except</em> for the <a data-cite="XML#sec-notation"><abbr title="Extended Backus-Naur Form">EBNF</abbr> productions</a> specified below, which replace the productions having the same number (if any) in the original grammar. Productions <a href="#rEmbTP">[174]</a> and following have been added and have no counterpart in the original grammar.</p>
+      <p>SPARQL-star is defined to follow the <a data-cite="SPARQL11-QUERY#sparqlGrammar">same grammar</a> as SPARQL, <em>except</em> for the <a data-cite="XML#sec-notation"><abbr title="Extended Backus-Naur Form">EBNF</abbr> productions</a> specified below, which replace the productions having the same number (if any) in the original grammar. Productions <a href="#rEmbTP">[174]</a> and following have been added and have no counterpart in the original grammar.</p>
```

Comment:
Minor: I'd leave the "1.1" because 1.0 grammar does not align with the 1.1 grammar numbering.

Reply:
Makes sense. I have restored the explicit reference to SPARQL 1.1 (see commit https://github.com/w3c/rdf-star/pull/115/commits/3be0fe024153143e48c663326b97a11cebd7c391)